sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitResultParser.java

```diff
@@ -238,7 +222,8 @@ private void saveTestsDetails(PhpUnitTestReport fileReport) {
     details.append("</tests-details>");
     InputFile unitTestFile = getUnitTestInputFile(fileReport);
     if (unitTestFile != null) {
-      context.saveMeasure(unitTestFile, new Measure(CoreMetrics.TEST_DATA, details.toString()));
+      context.<String>newMeasure().on(unitTestFile).withValue(details.toString()).forMetric(CoreMetrics.TEST_DATA).save();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "TEST_DATA"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)